### PR TITLE
Add default .npmrc

### PR DESCRIPTION
**Pull request by ascott1**
This adds a default `.npmrc` that will automatically save all npm installs to the package.json and pin the specific version installed. It also institutes that `.npmrc` for this repo.

## Review

@Scotchester @contolini

### CODE REVIEW

**Comment by Scotchester** [comment]
:+1: But you've got conflicts, now.

**Comment by ascott1** [comment]
The conflicts should be resolved now.

**Comment by jimmynotjim** [comment]
:+1:

**Comment by ascott1** [comment]
Is there anything holding this up? Seems like a good time to put it in action.

**Comment by contolini** [comment]
Just forgetfulness. :expressionless:

File: app/index.js [hunk 1]
```diff
@@ -243,6 +243,10 @@ var CapitalFrameworkGenerator = yeoman.generators.Base.extend({
         this.writeFileFromString( gitignore, '.gitignore' );
         // Kill the _cache dir.
         rimraf( this.destinationRoot() + '/_cache', done );
+    },
+
+    copyNpmrc: function() {
+      this.template('.npmrc', '.npmrc');
```
Comment (contolini) [inline app/index.js]: Change this to `this.copy` because we're not processing the file, just copying it over.

**Pull request merged**